src/pages/unauthenticated/PewPew.vue

```diff
@@ -265,23 +405,87 @@ import {
   mapTileLayer,
 } from '@/utils/map';
 import { HomeNavigation } from '@/components/home/SideNav';
-import Table from '@/components/Table';
 import { getQueryString } from '@/utils/urls';
 import BarChart from '@/components/charts/BarChart';
 import { Sprite, Texture, Graphics, utils as pixiUtils } from 'pixi.js';
 import Incident from '@/models/Incident';
-import { orderBy } from 'lodash';
+import OrganizationActivity from '@/components/OrganizationActivity.vue';
+import _, { orderBy } from 'lodash';
```

Comment:
No need to import `orderBy` like this. Just use `_.orderBy`
```suggestion
import _ from 'lodash';
```

Reply:
done